### DISCUSSION TITLE
Bug: Appearance saved banner says only "skin tone"

### DISCUSSION THREAD

**Discussion by unDeadHerbs** [community/community#22193] [Accessibility]
<p>The banner message for saving changes to Light/Dark theme says “Emoji skin tone preference successfully saved.”.  It should probably say “Appearance preference successfully saved.”.</p>

**Comment by unDeadHerbs** [comment]
<p>That, or the “Update preference” button applies only to the “skin tone” selection.  Which is ambiguous.</p>

**Comment by gaknoll** [comment, marked as answer]
<p>Hi <a class="mention" href="/undeadherbs">@unDeadHerbs</a> - thanks for taking the time to report this! The “update preference” button only applies to the emoji skin tone preference. We have fixing the emoji settings to also have an auto-save behavior that matches the theme setting in our backlog as we 100% agree this is confusing.</p>

**Comment by cehfisher** [comment]
Hi @unDeadHerbs - thank you for providing this feedback! After reviewing your issue report against the latest UI, this issue appears to be resolved now. Could you please confirm and close this issue? Thank you! (cc @queenofcorgis)
